package.json

```diff
@@ -4,7 +4,7 @@
   "description": "neu CLI for Neutralinojs",
   "main": "./bin/neu.js",
   "scripts": {
-    "test": "neu version"
+    "test": "node ./bin/neu.js"
```

Comment:
Hey.. please note that. We used `neu` before because we are testing the global installation here. You can check the `npm link` statement before the test statement [here](https://github.com/neutralinojs/neutralinojs-cli/blob/52cb49c39ca7274fb6f08bbece36091a1562be84/.github/workflows/ci.yml#L24)